File: Kattis/Avion.cpp
//Link:-https://open.kattis.com/problems/avion
#include <iostream>
#include <cstring>
#include <string>
using namespace std;
int main()
{
    int flag=1;
    for(int i=1;i<=5;i++)
    {
        string s;
        getline(cin,s);
        for(int i=0;i<=s.length()-2;i++)
        {
            if(s.substr(i,i+3)=="FBI")
            {
              flag=0;
              cout<<i<<" ";
            }
        }
    }
    if(flag==1)
     cout<<"HE GOT AWAY!"<<endl;
    return 0;
}

File: Kattis/Avion.java
import java.util.*;
import java.io.*;
import java.lang.*;
public class Avion {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        boolean flag = false;
        for(int i=0;i<5;i++)
        {
            String s = sc.nextLine();
            if(s.contains("FBI"))
            {
                flag = true;
               System.out.print(i+1+" ");
            }
        }
        if(!flag)
            System.out.println("HE GOT AWAY!");
    }
}
